### DISCUSSION TITLE
List of Bugs and Requests (by a_common_plant)

### DISCUSSION THREAD

**Discussion by EmilitiaEnnehrt** [Equinox-SS13/equinox-sojourn#55] [General]
> Just like with StarDust's list, the following is the requested (by me) bug/exploit list that has been known over the years by another regular codebase player.

> As well in this case, I'll be taking over maintenance operations of this, striking off the list as we go over and fix or verify them as being fixed or cannot be replicated. I've also changed the list around and categorize them for better priorty/viewing.

> Do know that some of these may be old bug or already fixed. Talk to the person to inquire more. 

> None of the content after this line is written by me, so all further questions and inquiry should go directly to the member; `a_common_plant`

---

## Bugs
- [ ] Satchels show on top of character when worn in suit slot of BS rig from armoury
> I would like to clarify for them that they meant the ENTIRE SPRITE, like the item sprite-- the WHOLE 32x32 sprite you see on the hand slot.
- [ ] Alot of circuit stuff, but a weird lizard is working on that on Soj
> Unsure who is fixing it, generally is a good idea to ask StarDust instead since they regularly use circuits (and is very knowledgable in it).
- [ ] Autofil from ID button in email app does not work
- [ ] Shotgun tubes (and probably alot of other things, but shotgun tubes are most important since coloured shells help you keep multiple tubes ready with different/mixed ammo) do not properly display colours of loaded ammunition into them.
> Putting this as bug not QoL, to explain more; loaded buckshot in a beanbag tube will display as beanbags in the tube.
- [ ] Ability to chose Advanced Styrete Sprite (we have couple of those for already existing styretes) or slap a colour wheel on it and make them colourable, currently there is only one avabile sprite making using multiple types of styretes annoying
> Actually moving this from QoL to bug, because sprites are supposed to work for this one but has worked and not functioned since. Same thing is reported as "unintended" by a former Sojourn developer; Shadow.
- [ ] in item/char descriptions `"` gets replaced with `&#34`; when you try to edit already existing description that has a `"`
- [x] some maskas/altyns especially in BS TAC storage start with their sprite mask down, but they have armour and protection as if mask up 
> Fix for Altyn is in for #53, however unsure about masks spawning with wrong sprite--personal experience is inconsistent (sometimes you need to interact twice to get a newly spawn helmet to push facemask up to where it's supposed to be)
- [ ] gps device actually working ???
> Right now it does not seem to work.

## Quality of Life Requests
- [ ] Ability to change sprites of mags or even ability to colour a magazine (colour wheel for the line on the sprite that signifies what ammo should be on the inside),possibly have 2 sprites, normal non marked sprite + one that is just that line in greyscale that will have colour applied to
- [ ] Ability to chose Advanced Styrete Sprite (we have couple of those for already existing styretes) or slap a colour wheel on it and make them colourable, currently there is only one avabile sprite making using multiple types of styretes annoying
- [ ] The ammo (and perhaps gun?) Disks from BS armoury perhaps should be in BS Tac storage instead, ammo ain't cheap and we can't expect ammo making job on duty every shift right now.
- [ ] Don't combine agony and normal DMG in weapon UI, make PVE damage if you really want for it
> Clarification, right now damage displayed in weapon information UI combine pain and normal damage together, making rubber display like "60 damage"
- [ ] give weapon UI a look over in general, bet some stats are missing
> Not sure what this means specifically, on top of my head they are things like projectile speed, deviation, size, and what slots are unavailable to mod in.
- [ ] A bigger thing but perhaps make gun frame part interaction make more sense? Do like how I can put a better grip on wooden 1911 and get a nicer gun, or changing the caliber for others
> ZenithEevee seems to be working on this?

## Feature Requests
- [x] gun parts/stats app? It was an idea thrown around on Sojourn some long time ago, guess ammo stats for it would be nice
> Suggesting--after the update--for a wikipage to be made for this under the name "Gun Manufacturing" or "Guide to Specialist/Armorer"
  - [ ] to extend the above, perhaps a stupid idea but a gun builder app, imagine simplified EFT gun modding interface and it spits out resulting stats + ability to save/load a file with it so you can email it as an order
- [ ] ability to send a same email to multiple people 
- [ ] look over computer apps, some unfortunately didn't receive Shadow's touch 
> I believe this is referring to TGUI update, as far as I know, Shadow is open for code bounty.
- [ ] camera/suit sensor app MAP, it shows empty 'space' currently + camera/sensor squares
> Toriate found out that this specifically requires the admin to compile every round..
- [x] combat balance/combat update rollback
> Personal "^^ THIS TBH"

**Comment by SigDoesCode** [comment]
The gun parts/stats is handled on this page of the wiki now: https://wiki.bluespace.engineer/en/guides/security-guides/Guns-and-Ammo
Similarly, the combat system and armor is being worked on. Checked both boxes off.
